### PR TITLE
GetRange

### PR DESCRIPTION
**Pull request by AradCarmi**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32930
## Description
- Created transformer to get range of values in an array
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/DeveloperTools/Scripts/GetRange/GetRange.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+args:
+- default: true
+  description: 'Comma separated list of indexes to slice by (0,1-3,6-7,...,etc).'
+  isArray: true
+  name: range
+  required: true
+  secret: false
+- default: true
+  description: 'List'
```
Comment (dantavori) [inline Packs/DeveloperTools/Scripts/GetRange/GetRange.yml:9]:
```suggestion
  description: 'The list to transform.'
```

File: Packs/DeveloperTools/Scripts/GetRange/GetRange.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  isArray: true
+  name: value
+  required: true
+  secret: false
+comment: 'Get a range of indexes from a list.'
```
Comment (dantavori) [inline Packs/DeveloperTools/Scripts/GetRange/GetRange.yml:14]:
```suggestion
comment: 'Gets specified indexes of a list.'
```

File: Packs/DeveloperTools/Scripts/GetRange/GetRange.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+args:
+- default: true
+  description: 'Comma separated list of indexes to slice by (0,1-3,6-7,...,etc).'
```
Comment (dantavori) [inline Packs/DeveloperTools/Scripts/GetRange/GetRange.yml:3]:
```suggestion
  description: 'A comma-separated list of indexes or ranges of to retrieve (e.g., 0,1-3,6-7).'
```

File: Packs/DeveloperTools/ReleaseNotes/1_2_15.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### New: GetRange
+- Get a range of indexes from a list. (Available from Cortex XSOAR 5.5.0)
+- Updated the Docker image to: *demisto/python3:3.10.1.25933*
```
Comment (dantavori) [inline Packs/DeveloperTools/ReleaseNotes/1_2_15.md:5]:
```suggestion
- Gets specified indexes of a list. (Available from Cortex XSOAR 5.5.0)
```

File: Packs/DeveloperTools/Scripts/GetRange/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+Get a range of indexes from a list.
```
Comment (dantavori) [inline Packs/DeveloperTools/Scripts/GetRange/README.md:1]: don't forget to regenerate. In addition, let's add an example.

**Review by dantavori — CHANGES_REQUESTED** [review]
looks great! let's try it out tomorrow before the demo.

File: Packs/DeveloperTools/Scripts/GetRange/GetRange_test.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+        When
+        - Calling main method.
+        Then
+        - Return filtered list.
+    """
+    from GetRange import main
+    import demistomock as demisto
+    with patch('GetRange.return_results') as return_results:
+        mocker.patch.object(demisto, 'args', return_value={'range': indexes, 'value': [i for i in range(10)]})
+        main()
```
[the remark this answers is not in the document]
Comment (AradCarmi) [reply Packs/DeveloperTools/Scripts/GetRange/GetRange_test.py:27]: I just called main method to cover as much as possible. But I understood from now on I would act that way.

**Review by dantavori — APPROVED** [review]

**Pull request merged**